overlay/base/usr/lib/system/_os/podman.py

```diff
@@ -526,3 +551,11 @@ def apply_delta(image: str, delta_image: str):
         xdelta3_proc.stdout.close()
         _ = podman_load_proc.communicate()
         _wait_for_processes(podman_load_proc, xdelta3_proc, podman_run_proc)
+
+
+def pull(image: str):  # pyright: ignore[reportUnusedParameter]
+    # TODO If we have a copy of the image locally,
+    # check to see if there is a _diff- tag that matches the digest for the
+    # Image to be pulled using image_digest, image_tags, and hex_to_base62
+    # If so, pull that instead, and then use apply_delta to apply it
+    pass
```

Comment:
@coderabbitai how would you implement this?

Reply:
How would you also do a size check to make sure that the delta is smaller than downloading the new image.
You can get the size of an image with `skopeo inspect docker://your-image:tag --raw | jq '[.layers[].size] | add' -r | numfmt --to=iec`

Reply:
image_size should be updated to handle the json parsing in the python code.
Instead of piping to `numfmt --to=iec`, this should just be done in the python code directly.